internal/persistence/postgres/transaction_writer.go

```diff
@@ -354,6 +355,7 @@ func (tw *transactionWriter) assignNonces(ctx context.Context, txInsertsByFrom m
 				}
 			}
 			log.L(ctx).Infof("Assigned nonce %s / %d to %s", signer, cacheEntry.nextNonce, op.txInsert.ID)
+			//nolint:gosec // Safe conversion as nextNonce is always positive
```

Comment:
Nit: This comment is inaccurate.

I agree with the ignore, but the justification is wrong (`uint64`-> `int64` cannot be "safe" due to positive - it can only be fully safe if you know all codepaths are < 2^63-1)